src/components/OrganizationAutocomplete.js

```diff
@@ -18,8 +20,17 @@ function renderInput(inputProps) {
 	const {classes, value, ref} = inputProps;
 	return (
 		<FormControl className={classes.textField}>
-			<InputLabel htmlFor="organization" className={classes.textFieldLabel}>
-				Organization Name
+			<InputLabel
+				htmlFor="organization"
+				className={
```

Comment:
use interpolated string

```suggestion
				className={ `${classes.textFieldLabel} ${classes.textAlignLeft} ${classes.colorGrey}` }
```